src/QsCompiler/BondSchemas/Argument/Argument.bond

```diff
@@ -1,7 +1,7 @@
 ﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-namespace Microsoft.Quantum.QsCompiler.BondSchemas.EntryPoint;
```

Comment:
I think it makes sense to have all the schemas in the same namespace. Having said that, maybe something like `Microsoft.Quantum.QsCompiler.BondSchemas.Execution`?